### ISSUE TITLE
Unable to load class 'kotlin.coroutines.Continuation'.

### ISSUE THREAD

**Issue by nek4life** [ktorio/ktor#1494]
When I run build project in intellij I get the following error after making a change to the ktor application.

Unable to load class 'kotlin.coroutines.Continuation'.

This is an unexpected error. Please file a bug containing the idea.log file.

This error continues to happen even when restarting the application until I run a gradle clean. Then restarting the application works as normal.

Below is my build.gradle.kts file. This is as generated by intellij.

```
import org.jetbrains.kotlin.gradle.dsl.Coroutines
import org.jetbrains.kotlin.gradle.tasks.KotlinCompile

val ktor_version: String by project
val kotlin_version: String by project
val logback_version: String by project

plugins {
    application
    kotlin("jvm") version "1.3.60"
}

group = "com.example"
version = "0.0.1"

application {
    mainClassName = "io.ktor.server.netty.EngineMain"
}

repositories {
    mavenLocal()
    jcenter()
    maven { url = uri("https://kotlin.bintray.com/ktor") }
}

dependencies {
    compile("org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version")
    compile("io.ktor:ktor-server-netty:$ktor_version")
    compile("ch.qos.logback:logback-classic:$logback_version")
    compile("io.ktor:ktor-server-core:$ktor_version")
    compile("io.ktor:ktor-html-builder:$ktor_version")
    compile("io.ktor:ktor-server-host-common:$ktor_version")
    compile("io.ktor:ktor-auth:$ktor_version")
    compile("io.ktor:ktor-jackson:$ktor_version")
    testCompile("io.ktor:ktor-server-tests:$ktor_version")
}

kotlin.sourceSets["main"].kotlin.srcDirs("src")
kotlin.sourceSets["test"].kotlin.srcDirs("test")

sourceSets["main"].resources.srcDirs("resources")
sourceSets["test"].resources.srcDirs("testresources")
```

**Comment by e5l** [comment]
@qwwdfsad

**Comment by cy6erGn0m** [comment]
What versions of Gradle and IDEA?

**Comment by nek4life** [comment]
I couldn't find the gradle path in intellij, but this is what I get from the terminal output. Please also find the relevant IDEA details below the gradle output.

gradle -v

------------------------------------------------------------
Gradle 5.1.1
------------------------------------------------------------

Build time:   2019-01-10 23:05:02 UTC
Revision:     3c9abb645fb83932c44e8610642393ad62116807

Kotlin DSL:   1.1.1
Kotlin:       1.3.11
Groovy:       2.5.4
Ant:          Apache Ant(TM) version 1.9.13 compiled on July 10 2018
JVM:          11.0.4 (AdoptOpenJDK 11.0.4+11)
OS:           Mac OS X 10.14.6 x86_64



IntelliJ IDEA 2019.3.1 (Ultimate Edition)
Build #IU-193.5662.31, built on December 10, 2019
Licensed to Charlie Choiniere
Subscription is active until May 15, 2020
Runtime version: 11.0.5+10-b520.17 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
macOS 10.14.6
GC: G1 Young Generation, G1 Old Generation
Memory: 2024M
Cores: 8
Registry: bigger.font.in.project.view=true, debugger.watches.in.variables=false, ide.balloon.shadow.size=0, js.debugger.webconsole=false
Non-Bundled Plugins: BashSupport, PsiViewer, Pythonid, com.andrey4623.rainbowcsv, com.binarysushi.sfcc-studio, com.google.gct.login, com.intellij.nativeDebug, org.jetbrains.kotlin, com.jetbrains.edu, com.jetbrains.lightThemePreview, com.jetbrains.plugins.ini4idea, com.sburlyaev.terminal.plugin, com.thvardhan.gradianto, intellij.prettierJS, io.ktor.intellij.plugin, net.ashald.envfile, net.seesharpsoft.intellij.plugins.csv, org.jetbrains.plugins.vagrant, org.jetbrains.plugins.vue, org.mule.tooling.intellij.raml, org.toml.lang, ru.adelf.idea.dotenv

**Comment by cy6erGn0m** [comment]
Could you please provide Kotlin plugin version (in IDEA) and the stacktrace?

**Comment by nek4life** [comment]
Execution failed for task ':compileKotlin'.
> kotlin/coroutines/Continuation

Unable to load class 'kotlin.coroutines.Continuation'.

This is an unexpected error. Please file a bug containing the idea.log file.

This is all I get. I did also try running with Stacktrace and that's the same message. Any idea on how to find the idea.log file?

**Comment by nek4life** [comment]
I found the log file using the JB Toolbox, but it doesn't appear to actually be creating log entries.

**Comment by cy6erGn0m** [comment]
@h0tk3y Any clue how do we capture enough data?

**Comment by h0tk3y** [comment]
A Gradle build scan might be useful. Otherwise, attaching a debugger to a Gradle daemon running with `-Dorg.gradle.debug=true` and inspecting the class loader of the plugin (or just printing the classpath from the script) could help as well.

**Comment by nek4life** [comment]
I ran with the debug flag and got the following stack trace. Removing `import io.ktor.client.features.auth.basic.*` from the generated project scaffold fixed the compile issue. I do have compile("io.ktor:ktor-auth:$ktor_version") in my gradle build file.


e: /Users/charliec/fun/bike-challenge/test/ApplicationTest.kt: (13, 32): Unresolved reference: auth
Task worker for ':': reuse connection
Task worker for ':': free connection (reuse = true)
Task worker for ':': reuse connection
Finished executing kotlin compiler using daemon strategy
Completing Build operation 'Execute execute for :compileTestKotlin'
Build operation 'Execute execute for :compileTestKotlin' completed
Fingerprinting property destinationDir (Output) for task ':compileTestKotlin'
Removed task artifact state for {} from context.
Finished executing task ':compileTestKotlin'
Completing Build operation 'Task :compileTestKotlin'
thread 91: dispatching class org.gradle.launcher.daemon.protocol.BuildEvent
Build operation 'Task :compileTestKotlin' completed

**Comment by fletcher91** [comment]
I just got this when the versions of kotlin and serialization didn't match, so there might be some downstream dependency using a different version?

```
plugins {
    application
    kotlin("jvm") version "1.3.61"
    id("org.jetbrains.kotlin.plugin.serialization") version "1.3.60" // setting to 1.3.61 fixed it
}
```

**Comment by v79** [comment]
This (suddenly) started happening to me when I started to write a test class. Gradle can build the project, and run the tests, but IntelliJ reports the same "Unable to load class 'kotlin.coroutines.Continuation'." error message.

Gradle version 4.10.2
IntelliJ 2019.3.1 Ultimate
Kotlin 1.3.61
Ktor 1.2.6

**Comment by oleg-larshin** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/KTOR-512">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.

**Comment by e5l** [comment]
Hey @nek4life, could you tell me if it's still a problem?

**Comment by nek4life** [comment]
Hey @e5l I unfortunately don't have the code for this anymore. It was on a laptop I don't have access to anymore and never created a repo for it as I was just exploring ktor for the first time.

**Comment by e5l** [comment]
Should be fixed with kotlin 1.3.61
